perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -619,7 +621,11 @@ def _Load(loader_index):
       results.append(self._Load(vms[loader_index], **kw))
       logging.info('VM %d (%s) finished', loader_index, vms[loader_index])
 
+    start = time.time()
     vm_util.RunThreaded(_Load, range(len(vms)))
+    events.record_event.send(
+        type(self).__name__, event='loader', start_timestamp=start,
```

Comment:
nit: the even for the run phase is "run" so I would assume the event here would be "load"